.github/workflows/test_install.yaml

```diff
@@ -0,0 +1,41 @@
+name: Test install
+on:
+  push:
+    branches:
+      - '**'
+
+jobs:
+  test-install:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        python-version: [ "3.9" ]
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+      - name: Check ips
+        run: ip a
+      - name: Ssh preparation
+        run: |
+          ssh-keygen -t rsa -b 4096 -N "" -f ~/.ssh/id_rsa -q
+          cp ~/.ssh/id_rsa.pub ~/.ssh/authorized_keys
+          ssh-keyscan -H 172.17.0.1 >> ~/.ssh/known_hosts
+      - name: Test ssh connection
+        run: ssh -i ~/.ssh/id_rsa 172.17.0.1 echo "Test"
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+      - name: Remove moby-runc to resolve conflicts
+        run: sudo apt remove moby-runc
+      - name: Check Iaas
+        continue-on-error: true
```

Comment:
pls preserve logs for future analysis after each procedure

Reply:
done